homeassistant/components/amberelectric/strings.json

```diff
@@ -1,25 +1,61 @@
 {
   "config": {
+    "error": {
+      "invalid_api_token": "[%key:common::config_flow::error::invalid_api_key%]",
+      "no_site": "No site provided",
+      "unknown_error": "[%key:common::config_flow::error::unknown%]"
+    },
     "step": {
+      "site": {
+        "data": {
+          "site_id": "Site NMI",
+          "site_name": "Site name"
+        },
+        "description": "Select the NMI of the site you would like to add"
+      },
       "user": {
         "data": {
           "api_token": "[%key:common::config_flow::data::api_token%]",
           "site_id": "Site ID"
         },
         "description": "Go to {api_url} to generate an API key"
-      },
-      "site": {
-        "data": {
-          "site_id": "Site NMI",
-          "site_name": "Site Name"
+      }
+    }
+  },
+  "services": {
+    "get_forecasts": {
+      "name": "Get price forecasts",
+      "description": "Retrieves price forecasts from Amber Electric for your property.",
+      "fields": {
+        "config_entry_id": {
+          "name": "Integration",
+          "description": "Which instance should we fetch forecasts from?"
```

Comment:
The description shouldn't be a question though, it should be explaining, just like the channel type

Reply:
How do I make this not "integration"?

```
ConfigEntrySelector({"integration": DOMAIN})
```

Because that sets the label helper of the field to "integration" which is weird.

<img width="1166" alt="image" src="https://github.com/user-attachments/assets/9a138c28-bfbe-42b2-9f04-c33bd8950ab9" />

Reply:
What you are changing here is this part
![image](https://github.com/user-attachments/assets/008fae0b-616a-4058-a714-bdf971cbdf4a)

Reply:
Got that.

But the placeholder on that text field probably shouldn't be "integration"?